bootstrap/README.md

```diff
@@ -61,8 +67,8 @@ Tests can be run in the following manner.
 
 ```
 $ npm install
-$ sudo node test/test.camera.js
-$ sudo node test/test.led.js
-$ sudo node test/test.servo.js
-$ sudo node test/test.speaker.js
+$ sudo node tests/test.camera.js
+$ sudo node tests/test.led.js
+$ sudo node tests/test.servo.js
+$ sudo node tests/test.speaker.js
```

Comment:
Removing the $ at the start of the above lines would make cut and paste much easier. To see this, go [here](https://github.com/ibmtjbot/tjbot/blob/feature/bb-bootstrap-initial/bootstrap/README.md#hardware-tests) and try to copy the five lines and paste them into terminal. Or double click a single line, copy it, and paste it to a terminal. The $ is also copied which is painful.

Removing the $ would also be more consistent with most of the other command shown in the README.